src/modules/visitor-space/components/VisitorSpaceNavigation/VisitorSpaceNavigation.tsx

```diff
@@ -111,11 +112,10 @@ const VisitorSpaceNavigation: FC<VisitorSpaceNavigationProps> = ({
 			<Navigation.Center title={title} />
 
 			<Navigation.Right placement="right">
-				{!isNil(accessEndDate) ? (
+				{!isNil(accessEndDate) && (
 					<span className="u-py-8 u-text-right">{accessEndDate}</span>
-				) : (
-					renderContact()
 				)}
+				{showContactInfo && renderContact()}
 			</Navigation.Right>
```

Comment:
OPTIONAL: We always want to render one or the other, never both at the same time
So maybe we should leave the tertiary:
```
			<Navigation.Right placement="right">
				{!isNil(accessEndDate) ? (
					<span className="u-py-8 u-text-right">{accessEndDate}</span>
				) : (
					showContactInfo && renderContact()
				)}
			</Navigation.Right>
```

Reply:
![image](https://user-images.githubusercontent.com/77958553/235877286-a9db40df-08d7-4062-bc06-69c65e916433.png)
Contact info has been removed from the design. That's why I've added the `showContactInfo` boolean. This way, we don't need to remove the functionality in case it's still needed on other places.